app/models/member.rb

```diff
@@ -32,6 +32,7 @@ class Member < ActiveRecord::Base
   scope :confirmed, -> { where('confirmed_at IS NOT NULL') }
   scope :located, -> { where("location <> '' and latitude IS NOT NULL and longitude IS NOT NULL") }
   scope :recently_signed_in, -> { reorder('updated_at DESC') }
+  scope :recently_joined, -> { reorder("created_at desc") }
```

Comment:
Hmmm. Unlike crops, I don't think there's much benefit to putting 
recently joined first.  I'd say alpha sort as default, for now.  In 
future we might want to have "most active" or something as the default, 
which would give you a more interesting first page.


-- 
Alex "Skud" Bayley
skud@growstuff.org
http://growstuff.org/